content/blog/dbplyr-2-0-0/index.Rmd

```diff
@@ -0,0 +1,204 @@
+---
+output: hugodown::hugo_document
+
+slug: dbplyr-2-0-0
+title: dbplyr 2.0.0
+date: 2020-11-03
+author: Hadley Wickham
+description: >
+    dbplyr 2.0.0 brings dplyr 1.0.0 compatibility, numeric improvements
+    to SQL translation (including new Amazon Redshift and SAP HANA 
+    backends), and an improved system for extending dbplyr to work with
+    other databases.
+    
+photo:
+  url: https://unsplash.com/photos/r2A6WYI8YIg
+  author: Shawn Ang
+
+categories: [package] 
+tags: [dbplyr, dplyr]
+---
+
+```{r, include = FALSE}
+library(dplyr, warn.conflicts = FALSE)
+```
+
+We're pleased to announce the release of [dbplyr](https://dbplyr.tidyverse.org/) 2.0.0.
+dbplyr is a database backend for [dplyr](https://dplyr.tidyverse.org/) that allows you to use a remote database as if it was a collection of local data frames: you write ordinary dplyr code and dbplyr translates it to SQL for you.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("dbplyr")
+```
+
+This blog post covers the major improvements in this version:
+
+-   dplyr 1.0.0 compatibility so you can now use `across()`, `relocate()`, `rename_with()`, and more.
+
+-   The major improvements to SQL translation.
+
+-   A snazzy new logo from [Allison Horst](https://www.allisonhorst.com).
+
+-   An improved extension system.
+
+Please see the [release notes](https://github.com/tidyverse/dbplyr/releases/tag/v2.0.0) for a full list of changes.
+
+```{r setup, message = FALSE}
+library(dbplyr)
+library(dplyr, warn.conflicts = FALSE)
+```
+
+## dplyr 1.0.0 compatibility
+
+dbplyr now supports all relevant features added in dplyr 1.0.0:
+
+-   `across()` is now translated into individual SQL statements.
+
+    ```{r}
+    lf <- lazy_frame(g = 1, a = 1, b = 2, c = 3)
+    lf %>% 
+      group_by(g) %>% 
+      summarise(across(everything(), mean, na.rm = TRUE))
+    ```
+
+-   `rename()` and `select()` support dplyr 1.0.0 tidyselect syntax, apart from predicate functions which can't easily work on computed queries.
```

Comment:
```suggestion
-   `rename()` and `select()` support dplyr tidyselect syntax, apart from predicate functions which can't easily work on computed queries.
```

Reply:
Since this is in a list of "relevant features added in dplyr 1.0.0", I don't think you need to reiterate the version number.